tests/topotests/ospf_metric_propagation/r1/show_ip_route-1.json

```diff
@@ -4,7 +4,7 @@
       "prefix":"10.0.94.0/24",
       "prefixLen":24,
       "protocol":"bgp",
-      "vrfId":9,
+      "vrfId":"*",
```

Comment:
Why not just leave the key out for the comparison? Are you changing the comparison to an exact match?